ocrd_network/ocrd_network/logging.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+from ocrd_utils import safe_filename
+
+
+OCRD_NETWORK_MODULES = [
+    "mets_servers",
+    "processing_jobs",
+    "processing_servers",
+    "processing_workers",
+    "processor_servers"
+]
+
+
+def get_root_logging_dir(module_name: str) -> Path:
+    if module_name not in OCRD_NETWORK_MODULES:
+        raise ValueError(f"Invalid module name: {module_name}, should be one of: {OCRD_NETWORK_MODULES}")
+    # TODO: Utilize env variable to set the root
+    module_log_dir = Path("/tmp", "ocrd_network_logs", module_name)
+    module_log_dir.mkdir(parents=True, exist_ok=True)
+    return module_log_dir
+
+
+def get_cache_locked_pages_logging_file_path() -> Path:
+    return get_root_logging_dir("processing_servers") / "cache_locked_pages.log"
```

Comment:
I think this should use the constants defined instead of hard coded. I know this is checked in `get_root_logging_dir` but still. Other methods as well.

Reply:
> I think this should use the constants defined

Which defined constants? There's just a defined list with the modules. Do you mean we need an enum here?

Reply:
You are right, probably I thought it was an enum, my bad. I would keep it then as it is now.